src/nfs/nfs_client_impl.cpp

```diff
@@ -274,11 +275,13 @@ void nfs_client_impl::continue_copy()
             zauto_lock l(req->lock);
             const user_request_ptr &ureq = req->file_ctx->user_req;
             if (req->is_valid) {
-                _copy_token_buckets->get_token_bucket(ureq->file_size_req.dest_disk_tag)
-                    ->consumeWithBorrowAndWait(req->size,
-                                               FLAGS_max_copy_rate_megabytes_per_disk << 20,
-                                               1.5 *
-                                                   (FLAGS_max_copy_rate_megabytes_per_disk << 20));
+                if (FLAGS_max_copy_rate_megabytes_per_disk) {
```

Comment:
```if (FLAGS_max_copy_rate_megabytes_per_disk > 0)```

Reply:
why FLAGS_max_copy_rate_megabytes_per_disk > 0 is better?

Reply:
1. It's better to use `>` for int type. 
2. FLAGS_max_copy_rate_megabytes_per_disk is int_32, may smaller than 0